definitions/infra-unix_host/definition.yml

```diff
@@ -0,0 +1,8 @@
+domain: INFRA
+type: UNIX_HOST
+name: hostname
+identifier: hostname
+
+compositeMetrics:
+  goldenMetrics: ./golden_metrics.yml
+  summaryMetrics: ./summary_metrics.yml
```

Comment:
Sorry for the changes, since the schema is still evolving, but the PR is failing because:

- These fields need to be under `synthesis` as in [this example](https://github.com/newrelic-experimental/entity-synthesis-definitions/blob/main/docs/example-entity-definition.yml).
- The golden and summary metrics are now arrays.

Reply:
understood; updates applied, thanks!

Reply:
@IreneMLC - still failing; no rush, just FYI

Reply:
Thanks for adding the changes! the validator seems to be complaining about the yaml format.

Reply:
The issue is comming from the field in backticks so you can surround it with single quotes and it should be good to go:
'\`unixMonitor:Disk\`'

Reply:
checks are passing, thanks for all of the help!

Reply:
Thanks for adding the changes!